extendr-api/src/wrapper/dataframe.rs

```diff
@@ -7,7 +7,25 @@ pub trait IntoDataFrameRow<T> {
 #[derive(PartialEq, Clone)]
 pub struct Dataframe<T> {
     pub(crate) robj: Robj,
-    marker: std::marker::PhantomData<T>,
+    _marker: std::marker::PhantomData<T>,
+}
+
+impl<T> From<Dataframe<T>> for Robj {
+    fn from(value: Dataframe<T>) -> Self {
+        value.robj
+    }
+}
+
+impl<T> FromRobj<'_> for Dataframe<T> {
```

Comment:
Can you delegate this impl to `TryFrom`?

Reply:
It is already available for TryFrom. I'm expanding the compatibility, until we drop the big Only-TryFrom PR.

Reply:
I meant you duplicate coed here and in TryFrom

Reply:
Good hint. Hold on.